package.json

```diff
@@ -131,7 +132,8 @@
     "storybook-addon-theme-playground": "^1.2.0",
     "style-loader": "^2.0.0",
     "terser-webpack-plugin": "^4.2.3",
-    "webpack-cli": "^4.4.0",
+    "webpack-cli": "^4.9.2",
```

Comment:
Bumped this version because of https://github.com/symfony/webpack-encore/issues/1069#issue-1093112041 .